com.vrcfury.vrcfury/Editor/VF/Feature/ToggleBuilder.cs

```diff
@@ -119,7 +142,7 @@ public class ToggleBuilder : FeatureBuilder<Toggle> {
             var numParam = fx.NewInt(paramName, synced: true, saved: model.saved, def: model.defaultOn ? 1 : 0, usePrefix: usePrefixOnParam);
             onCase = numParam.IsNotEqualTo(0);
         } else {
-            var boolParam = fx.NewBool(paramName, synced: true, saved: model.saved, def: model.defaultOn, usePrefix: usePrefixOnParam);
+            var boolParam = fx.NewBool(paramName, synced: addMenuItem, saved: model.saved, def: model.defaultOn, usePrefix: usePrefixOnParam);

```

Comment:
This change is unsafe for people using globalParam for its original purpose, which is to be able to drive the toggle on and off from another source. If that other source isn't synchronized, the local and remote state of this toggle can get out of sync.

Reply:
I changed this to make it non-synced only if it's a vrc built-in.